topics/tutorials/file_and_code_templates/providing_file_templates.md

```diff
@@ -0,0 +1,97 @@
+[//]: # (title: Providing File and Code Templates)
+
+<!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+The IntelliJ Platform allows plugins to provide custom file templates specific to the plugin's functionalities.
+In order to include custom templates in the plugin, a template file has to be created and placed in the specific place of plugin resources, depending on the template's purpose.
+
+## Creating File Templates
+
+There are multiple ways of creating file templates:
+* [Use "File | Save File as Template…" action](https://www.jetbrains.com/help/idea/using-file-and-code-templates.html#save-file-as-template)
+* [Create a new file template](https://www.jetbrains.com/help/idea/using-file-and-code-templates.html#create-new-template)
+* [Copy an existing file template](https://www.jetbrains.com/help/idea/using-file-and-code-templates.html#copy-existing-template)
+
+Once the file templates are created and saved, they should be copied to the plugin project's <path>resources</path> directory.
+The created template can be found in the [IDE configuration directory](https://www.jetbrains.com/help/idea/directories-used-by-the-ide-to-store-settings-caches-plugins-and-logs.html#config-directory) in the <path>fileTemplates</path> directory, or they can be exported via <menupath>File | Manage IDE Settings | Export Settings</menupath> by selecting the <control>File Templates (schemes)</control> checkbox.
+The exported ZIP file will contain the <path>fileTemplates</path> directory with the created templates.
+The <path>fileTemplates</path> directory should be moved to the plugin project's <path>resources</path> folder, and the `.ft` extension must be added to the template files, e.g., `My Class.java` must be renamed to `My Class.java.ft`.
```

Comment:
all filenames must be written using  `<path>` everywhere
`MyClass.java`->`<path>MyClass.java</path>` etc

Reply:
Fixed, thanks.